link/kprobe.go

```diff
@@ -230,6 +230,11 @@ func pmuProbe(typ probeType, symbol, path string, offset uint64, ret bool) (*per
 	if errors.Is(err, os.ErrNotExist) || errors.Is(err, unix.EINVAL) {
 		return nil, fmt.Errorf("symbol '%s' not found: %w", symbol, os.ErrNotExist)
 	}
+	// Since commit cb9a19fe4aa51, ENOTSUPP is returned when attempting to set
```

Comment:
I think the specific commit is just a small refactor, the `is_swbp_insn(...)` check was already there so I'd remove the first part of the comment

Reply:
I thought a sha would be more convenient than a path or a function name, since those tend to change over time. I'll specify 'at least' there. :)